pkg/services/provisioning/datasources/testdata/multiple-org-default/config.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: 1
+
+datasources:
+  - orgId: 1
+    name: prometheus
+    type: prometheus
+    isDefault: True
+    access: proxy
+    url: http://prometheus.example.com:9090
+  - name: Graphite
+    type: graphite
+    access: proxy
+    url: http://localhost:8080
+    is_default: true
```

Comment:
Looking at this test data it looks like it shouldn't work (multiple default dashboards for the same org), but it does work as only some of them has the field `isDefault`, while half has `is_default` which I don't think will work.

Reply:
Good catch :+1: is_default should be removed

Reply:
Seems like is_default was the v0 syntax and isDefault is the v1 syntax. Very confusing :smile: